app/models/event.rb

```diff
@@ -0,0 +1,7 @@
+class Event < ActiveRecord::Base
+  has_many :lists
+
+  validates :title, presence: true
```

Comment:
I didn't want to add any other constraints (e.g. validating that the start date is in the future, validating a presence for description) on my own since that involves making a lot of assumptions about use cases. A few that could possibly be added:
* validate that the end date is in the future
* validate that the start date exists
* validate that the end date exists

but I'm not sure how necessary they are, or if there are use cases outside of those